src/analysis/Environments/Environments.test.ts

```diff
@@ -602,30 +599,37 @@ describe('Environments', () => {
       const buttons1 = getAllByRole(app1ButtonsCell, 'button');
 
       // Assert
-      expect(buttons1.length).toBe(2);
-      expect(buttons1[0].textContent).toBe('Pause');
+      // TODO: Back to 2 / Pause once https://broadworkbench.atlassian.net/browse/PROD-905 is resolved
```

Comment:
Hmm... this feel wrong. Why does pause stop showing up for azure apps (which have `appType` `Cromwell`)? I thought your change should only affect apps with `appType` `Galaxy` ?

Reply:
Well technically we are not supporting pausing for Azure app types either so I disabled it as well.

Reply:
This is not what this test is testing. This test is working double duty and testing both the visual layer and the provider layer. It is ensuring that, when I click the pause button for this app, the proper code path (in this case an unimplemented one) is hit. Thus, when we do support it, it will already be tested

Reply:
In other words, I'm fine with disabling pause for azure, but the pause button being enabled for azure should not be coverage we lose

Reply:
🤔 I think I understand what you are saying, but I might need some handholing to untangle that test if you don't mind :)

Reply:
Well not really because if we decide to add the button back then we will have 2 objects in the array to test